src/client/js/components/Admin/SlackIntegration/SlackIntegration.jsx

```diff
@@ -143,8 +143,8 @@ const SlackIntegration = (props) => {
 
         {t('admin:slack_integration.selecting_bot_types.selecting_bot_type')}
 
-        <div className="row my-4">
-          <div className="card-deck mx-auto">
+        <div className="my-4">
+          <div className="row flex-wrap-reverse justify-content-center">
             {botTypes.map((botType) => {
               return (
                 <BotTypeCard
```

Comment:
```suggestion
<div className="m-3">
　<BotTypeCard ...
</div>
```
BotTypeCard を扱う親コンポーネントで margin の規定